website/docs/reference/resource-configs/alias.md

```diff
@@ -25,9 +62,8 @@ To learn more about changing the way that dbt generates a relation's `identifier
 ## Usage
 
 ### Seeds
-Configure a seed's alias in your `dbt_project.yml` file.
 
-The seed at `seeds/country_codes.csv` will be built as a <Term id="table" /> named `country_mappings`.
+In this example, the seed at `seeds/country_codes.csv` will be built as a <Term id="table" /> named `country_mappings`.
```

Comment:
Although it is useful to make clear that the seed lives at `seeds/country_codes.csv` so maybe that should migrate up to the example file before deletion